tests/reference_tests/reference_test.py

```diff
@@ -527,6 +527,16 @@ def test_nt(line, folder):
         traj_data=trajectory,
         measurement_line=line,
     )
+
+    # In JPSreport crossing was counted when a pedestrians touches the
```

Comment:
At this point I'm not sure if references to JPSreport are helpful. 
Optionally, consider remove this reference to JPSreport

Reply:
I will keep the reference as currently, JPSreport is still our "ground-truth"